examples/CSharp/Tutorial 3.2 - Diffuse Lighting/MathHelper.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Tutorial
+{
+    public static class MathHelper
+    {
+        public static float DegreesToRadians(double degrees)
+        {
+            return (float)(Math.PI / 180f * degrees);
+        }
```

Comment:
Weird that this returns `float` but takes `double`? Any reason for that?

Reply:
My mistake, it was originally a double in and out then converted to float, refactoring left it in a weird state so I will fix that